deploy/.env.ingest.production.local.tpl

```diff
@@ -88,7 +88,11 @@ PREFIX="${TF_WORKSPACE}-${TF_VAR_app}"
   },
   "Peering": {
     "Peers": null
-  }
+  },
+  "OTELServiceName": "$PREFIX",
+  "OTELExporterEndpoint": "https://api.honeycomb.io:443",
+  "OTELExporterHeaders": "x-honeycomb-team=$HONEYCOMB_API_KEY",
+  "OTELSamplerRatio": 0.0001
```

Comment:
to make this depend on the environment, you can add this right before doing the read:
```
if [ "$TF_WORKSPACE" == "prod" ]; then
  BASE_TRACE_SAMPLE_RATIO="0.0001"
else
  BASE_TRACE_SAMPLE_RATIO="1.0"
fi
```
and then set this property to that var. I do an equivalent of this in the indexing-service.